scripts/mytop.sh

```diff
@@ -919,10 +999,15 @@ sub GetData()
             }
         }
 
-	open L, "</proc/loadavg";
-	my $l = <L>;
-	close L;
-	chomp $l;
+        my $l;
+        if (-e "/proc/loadavg")
+        {
+            ## To avoid warnings if the OS is not Linux
+            open (my $fh, "</proc/loadavg");
```

Comment:
I would use `<` mode as a second parameter as stated in [doc](https://perldoc.perl.org/functions/open.html).